examples/server/server.c

```diff
@@ -2367,7 +2367,7 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 #endif
 
         if (echoData == 0 && throughput == 0) {
-            ServerRead(ssl, input, sizeof(input)-1);
+            ServerRead(ssl, input, (int) XSTRLEN(input));
```

Comment:
This seems wrong, the old code was correct. The last parameter is the size of the output buffer, not whatever is in the buffer (strlen).